src/main/java/com/ixi_U/user/controller/SubscribedController.java

```diff
@@ -0,0 +1,30 @@
+package com.ixi_U.user.controller;
+
+import com.ixi_U.user.dto.CreateSubscribedRequest;
+import com.ixi_U.user.service.SubscribedService;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/subscribed")
+public class SubscribedController {
+
+    private final SubscribedService subscribedService;
+
+    public SubscribedController(SubscribedService subscribedService) {
+        this.subscribedService = subscribedService;
+    }
+
+    @PostMapping("/{userId}")
+    public ResponseEntity<Void> updateSubscribed(
+            @PathVariable("userId") String userId,
+            @RequestBody @Valid CreateSubscribedRequest request) {
+        subscribedService.updateSubscribed(userId, request);
+        return ResponseEntity.ok().build();
```

Comment:
```suggestion

        return ResponseEntity.ok().build();
```
앞선 리뷰들과 같은 맥락에서 코드 컨벤션 고려하여 수정해주세요

Reply:
수정했습니다.